app/src/main/java/org/simple/clinic/contactpatient/views/SetAppointmentReminderView.kt

```diff
@@ -4,16 +4,35 @@ import android.content.Context
 import android.util.AttributeSet
 import android.view.View
 import androidx.constraintlayout.widget.ConstraintLayout
+import kotlinx.android.synthetic.main.contactpatient_appointmentreminder.view.*
 import org.simple.clinic.R
 
+private typealias PreviousStepperClicked = () -> Unit
+private typealias NextStepperClicked = () -> Unit
+private typealias AppointmentDateClicked = () -> Unit
+private typealias DoneClicked = () -> Unit
+
 class SetAppointmentReminderView(
     context: Context,
     attributeSet: AttributeSet
 ) : ConstraintLayout(context, attributeSet) {
 
+  var previousStepperClicked: PreviousStepperClicked? = null
```

Comment:
Can we rename this to `decrementDateClicks` and `DecrementStepperClicked`?

Reply:
Done.